### ISSUE TITLE
JAWS unable to read the list item count in an unordered list

### ISSUE THREAD

**Issue by anuvenkatesh1** [FreedomScientific/VFO-standards-support#639]
### Summary - As we navigate through an unordered HTML list of buttons/links etc, why is JAWS not announcing the position count of the focused list item?

> Try the below HTML code with a list of buttons with JAWS (I tried 2020, 2019, 2022. same behavior observed)
> 
> Example:
<!DOCTYPE html>
<html>
<body>

<h2>An unordered HTML list</h2>

<ul>
  <li><button>Test 1</button></li>
  <li><button>Test 2</button></li>
  <li><button>Test 3</button></li>
</ul>  

</body>
</html>

### Expected result

I was expecting JAWS to announce the list item's position count eg: Button 2 - 2 of 3, Button 3 - 3 of 3. However this didn't get announced either in forms or in VPC mode.

### Actual result

![image](https://user-images.githubusercontent.com/32471459/168935461-24899573-6bce-40bf-91d9-6f8f6a502461.png)

#### JAWS version and build number
![image](https://user-images.githubusercontent.com/32471459/168935312-b09b2a91-1667-41b2-b6e4-9bcf4f344172.png)

 
#### Operating System and version
Windows 10

#### Browser and version:

Chrome 101
